pkg/client/telemetry/telemetry.go

```diff
@@ -0,0 +1,28 @@
+// Package telemetry ...
+package telemetry
+
+import (
+	"github.com/spf13/pflag"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+)
+
+// TelemetryConfig is a wrapper for the telemetry configuration.
+type TelemetryConfig struct {
+	phonehome.Config
+}
+
+// NewTelemetryConfig creates a new telemetry configuration.
+func NewTelemetryConfig() *TelemetryConfig {
+	return &TelemetryConfig{}
```

Comment:
Nit:
This would add some context to all messages issued from this client:
```suggestion
	return &TelemetryConfig{
		ClientID:   getFleetManagerID(),
		ClientName: "ACS Fleet Manager",
	}
```

Reply:
Do we need an `Identify` call as well for fleet manager?

Reply:
If only there are any fleet manager specific properties (like version) that could be of our interest. Note that any Identity properties will have effect only on the following Track events on Amplitude.

Reply:
We don't expose the build info in fleet manager at the moment. Let's add this later on if need be.